app/helpers/originality_reports_helper.rb

```diff
@@ -19,6 +19,12 @@ def originality_report(score)
     }
   end
 
+  def originality_report_edit_url
+    base_url = tool_proxy_from_assignment.base_tc_url
+    "#{base_url}/api/lti/assignments/#{params['assignment_tc_id']}"\
```

Comment:
We should be able to get the URLs from the tcp at registration time and store them on the tool proxy (similar to what we do with the `authorization_url`).